data/scrape/lib/codes.py

```diff
@@ -174,7 +174,14 @@ def main() -> None:
             wiktionary_code = _scrape_wiktionary_language_code(
                 wiktionary_name.replace(" ", "_")
             )
-            iso639_lang = iso639.Language.match(wiktionary_code)
+
```

Comment:
Check looks good. Don't you need some kind of `continue` statement to exit the loop, or a default value for `iso_639_lang`? Otherwise if you hit the `except` clause, it looks to me like the identifier `iso639_lang` will be undefined (or take the value of the previous loop given Python's scoping rules). But that's just my conjecture from reading the function...I may be wrong.

Reply:
This is a good catch, I'll fix it.

This actually hadn't caused an error in the loop when I tested the program, because it was taking on the value of iso639_lang from the previous iteration. Glad you noticed it

Reply:
Yeah, things like that are why I hate Python's weird scope rules...